services/src/test/scala/cromwell/services/metadata/impl/pubsub/PubSubMetadataServiceActorSpec.scala

```diff
@@ -1,274 +0,0 @@
-package cromwell.services.metadata.impl.pubsub
-
-import java.time.OffsetDateTime
-
-import akka.actor.{ActorInitializationException, ActorRef, Props}
-import akka.testkit.{EventFilter, TestProbe}
-import com.google.api.services.pubsub.model.Topic
-import com.typesafe.config.{Config, ConfigFactory}
-import cromwell.core.WorkflowId
-import cromwell.services.ServicesSpec
-import cromwell.services.metadata.MetadataService.{
-  MetadataWriteFailure,
-  MetadataWriteSuccess,
-  PutMetadataAction,
-  PutMetadataActionAndRespond
-}
-import cromwell.services.metadata.{MetadataEvent, MetadataKey, MetadataValue}
-import org.broadinstitute.dsde.workbench.google.GooglePubSubDAO
```

Comment:
Does this let us stop building workbench-libs into this jar? (I know it's still needed for CromIAM, pact, etc)

Reply:
Good call, I'll check it out.

Reply:
I'm running into some annoying SBT conflicts after ~1hr so I'm going to punt on this for now.